### ISSUE TITLE
extract correct number of studies and correct number of intervention

### ISSUE THREAD

**Issue by softloud** [softloud/happypillpain#11]
![image](https://user-images.githubusercontent.com/13408706/115034659-a8a30780-9ec3-11eb-8946-e172b2036aae.png)

hollie's counts == charles' counts for study 

- [ ] sub pain
- [ ] mod pain
- [x] pain
- [x] sleep

**Comment by softloud** [comment]
- [ ] @holliebirkinshaw to do own count for at least one for q&a

**Comment by softloud** [comment]
## substantial pain current studies

```
> models %>% keep(~ .$outcome == "substantial pain") %>% pluck(1, "network", "agd_arm") %>% select(.study, comments) %>% distinct()
# A tibble: 4 x 2
  .study       comments                                                                                         
  <fct>        <chr>                                                                                            
1 Wang 2017    Title: Efficacy and safety of duloxetine in Chinese patients with chronic pain due to osteoarthr…
2 Lipone 2020  Title: Efficacy and Safety of Low Doses of Trazodone in Patients Affected by Painful Diabetic Ne…
3 Loldrup 1989 Title: Clomipramine and mianserin in chronic idiopathic pain syndrome. A placebo controlled study
4 Uchio 2018   Title: A randomized, double-blind, placebo-controlled phase III trial of duloxetine in Japanese …
```

**Comment by HollieBirkinshaw** [comment]
[19.04.21.csv](https://github.com/softloud/happypillpain/files/6335950/19.04.21.csv)
Substantial (defined as ≥50%) pain relief

**Comment by softloud** [comment]
[1] "combined_visual_analogue_scales_0_500_in_total_for_pain_paraesthesia_dysaesthesia_nightly_aggravation_and_sleep_disturbances_baseline_median"             
  [2] "combined_visual_analogue_scales_0_500_in_total_for_pain_paraesthesia_dysaesthesia_nightly_aggravation_and_sleep_disturbances_baseline_n"                  
  [3] "combined_visual_analogue_scales_0_500_in_total_for_pain_paraesthesia_dysaesthesia_nightly_aggravation_and_sleep_disturbances_post_intervention_median"    
  [4] "combined_visual_analogue_scales_0_500_in_total_for_pain_paraesthesia_dysaesthesia_nightly_aggravation_and_sleep_disturbances_post_intervention_n"         
  [5] "good_pain_relief_50_percent_relief_endpoint_percent"                                                                                                      
  [6] "good_pain_relief_50_percent_relief_endpoint_n"                                                                                                            
  [7] "improvement_reduction_in_median_pain_score_vas_25_50_percent_moderate_response_percent"                                                                   
  [8] "improvement_reduction_in_median_pain_score_vas_25_50_percent_moderate_response_n"                                                                         
  [9] "improvement_reduction_in_median_pain_score_vas_50_percent_good_response_percent"                                                                          
 [10] "improvement_reduction_in_median_pain_score_vas_50_percent_good_response_n"                                                                                
 [11] "number_of_responders_50_percent_improvement_in_brief_pain_inventory_average_pain_bocf_endpoint_percent"                                                   
 [12] "number_of_responders_50_percent_improvement_in_brief_pain_inventory_average_pain_bocf_endpoint_n"                                                         
 [13] "pain_50_30_percent_decrease_from_baseline_in_average_pain_score_endpoint_percent"                                                                         
 [14] "pain_50_30_percent_decrease_from_baseline_in_average_pain_score_endpoint_n"                                                                               
 [15] "pain_50_percent_decrease_from_baseline_in_average_24hr_pain_score_endpoint_percent"                                                                       
 [16] "pain_50_percent_decrease_from_baseline_in_average_24hr_pain_score_endpoint_n"                                                                             
 [17] "pain_50_percent_decrease_from_baseline_in_average_pain_score_baseline_percent"                                                                            
 [18] "pain_50_percent_decrease_from_baseline_in_average_pain_score_baseline_n"                                                                                  
 [19] "pain_50_percent_decrease_from_baseline_in_average_pain_score_endpoint_percent"                                                                            
 [20] "pain_50_percent_decrease_from_baseline_in_average_pain_score_endpoint_n"                                                                                  
 [21] "pain_50_percent_decrease_from_baseline_in_average_pain_score_bocf_endpoint_percent"                                                                       
 [22] "pain_50_percent_decrease_from_baseline_in_average_pain_score_bocf_endpoint_n"                                                                             
 [23] "pain_50_percent_decrease_from_baseline_in_average_pain_score_locf_endpoint_percent"                                                                       
 [24] "pain_50_percent_decrease_from_baseline_in_average_pain_score_locf_endpoint_n"                                                                             
 [25] "pain_50_percent_decrease_from_baseline_in_mean_pain_score_endpoint_percent"                                                                               
 [26] "pain_50_percent_decrease_from_baseline_in_mean_pain_score_endpoint_n"                                                                                     
 [27] "pain_50_percent_decrease_from_baseline_typical_pain_intensity_post_intervention_grouped_for_venlafaxine_altogether_not_by_dose_percent"                   
 [28] "pain_50_percent_decrease_from_baseline_typical_pain_intensity_post_intervention_grouped_for_venlafaxine_altogether_not_by_dose_n"                         
 [29] "pain_50_percent_decrease_from_baseline_vas_pain_score_completers_post_intervention_n"                                                                     
 [30] "pain_50_percent_decrease_from_baseline_vas_pain_score_completers_post_intervention_percent"                                                               
 [31] "pain_50_percent_decrease_from_baseline_vas_pain_score_completers_post_intervention_confidence_limits"                                                     
 [32] "pain_50_percent_decrease_from_baseline_vas_pain_score_endpoint_post_intervention_n"                                                                       
 [33] "pain_50_percent_decrease_from_baseline_vas_pain_score_endpoint_post_intervention_percent"                                                                 
 [34] "pain_50_percent_decrease_from_baseline_vas_pain_score_endpoint_post_intervention_confidence_limits"                                                       
 [35] "pain_50_percent_decrease_in_average_pain_severity_in_bpi_endpoint_percent"                                                                                
 [36] "pain_50_percent_decrease_in_average_pain_severity_in_bpi_endpoint_n"                                                                                      
 [37] "pain_50_percent_decrease_in_bpi_24_hour_average_pain_score_baseline_percent"                                                                              
 [38] "pain_50_percent_decrease_in_bpi_24_hour_average_pain_score_baseline_n"                                                                                    
 [39] "pain_50_percent_decrease_in_bpi_24_hour_average_pain_score_endpoint_percent"                                                                              
 [40] "pain_50_percent_decrease_in_bpi_24_hour_average_pain_score_endpoint_n"                                                                                    
 [41] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_baseline_percent"                                                                           
 [42] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_baseline_n"                                                                                 
 [43] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_endpoint_percent"                                                                           
 [44] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_endpoint_n"                                                                                 
 [45] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_patient_diaries_baseline_percent"                                                           
 [46] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_patient_diaries_baseline_n"                                                                 
 [47] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_patient_diaries_endpoint_percent"                                                           
 [48] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_patient_diaries_endpoint_n"                                                                 
 [49] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_patient_diaries_post_intervention_percent"                                                  
 [50] "pain_50_percent_decrease_in_weekly_mean_24_hour_pain_severity_patient_diaries_post_intervention_n"                                                        
 [51] "pain_50_percent_or_greater_reduction_from_baseline_in_nrs_pain_score_responders_endpoint_percent"                                                         
 [52] "pain_50_percent_or_greater_reduction_from_baseline_in_nrs_pain_score_responders_endpoint_n"                                                               
 [53] "pain_50_percent_reduction_in_paper_gracely_pain_scores_4_0_unit_decrease_endpoint_percent"                                                                
 [54] "pain_50_percent_reduction_in_paper_gracely_pain_scores_4_0_unit_decrease_endpoint_n"                                                                      
 [55] "pain_50_percent_reduction_in_vas_0_10_endpoint_percent"                                                                                                   
 [56] "pain_50_percent_reduction_in_vas_0_10_endpoint_n"                                                                                                         
 [57] "pain_50_percent_reduction_of_weekly_mean_score_in_24_hour_average_pain_severity_ratings_locf_endpoint_percent"                                            
 [58] "pain_50_percent_reduction_of_weekly_mean_score_in_24_hour_average_pain_severity_ratings_locf_endpoint_n"                                                  
 [59] "pain_bpi_average_pain_severity_score_50_percent_decrease_from_baseline_to_endpoint_endpoint_percent"                                                      
 [60] "pain_bpi_average_pain_severity_score_50_percent_decrease_from_baseline_to_endpoint_endpoint_n"                                                            
 [61] "pain_bpi_average_pain_severity_score_50_percent_decrease_from_baseline_to_endpoint_post_intervention_percent"                                             
 [62] "pain_bpi_average_pain_severity_score_50_percent_decrease_from_baseline_to_endpoint_post_intervention_n"                                                   
 [63] "pain_chest_pain_severity_symptom_intensity_scores_participants_with_50_percent_50_percent_or_complete_response_50_percent_percent"                        
 [64] "pain_chest_pain_severity_symptom_intensity_scores_participants_with_50_percent_50_percent_or_complete_response_50_percent_n"                              
 [65] "pain_chest_pain_severity_symptom_intensity_scores_participants_with_50_percent_50_percent_or_complete_response_50_percent_percent_2"                      
 [66] "pain_chest_pain_severity_symptom_intensity_scores_participants_with_50_percent_50_percent_or_complete_response_50_percent_n_2"                            
 [67] "pain_chest_pain_severity_symptom_intensity_scores_participants_with_50_percent_50_percent_or_complete_response_complete_response_percent"                 
 [68] "pain_chest_pain_severity_symptom_intensity_scores_participants_with_50_percent_50_percent_or_complete_response_complete_response_n"                       
 [69] "pain_improvement_in_vas_good_more_than_50_percent_percent"                                                                                                
 [70] "pain_improvement_in_vas_good_more_than_50_percent_n"                                                                                                      
 [71] "pain_improvement_in_vas_moderate_25_50_percent_percent"                                                                                                   
 [72] "pain_improvement_in_vas_moderate_25_50_percent_n"                                                                                                         
 [73] "pain_intensity_participants_with_reduction_of_50_percent_from_baseline_in_vas_pi_scores_endpoint_percent"                                                 
 [74] "pain_intensity_participants_with_reduction_of_50_percent_from_baseline_in_vas_pi_scores_endpoint_n"                                                       
 [75] "pain_participants_with_50_percent_decrease_from_baseline_in_average_24hr_pain_score_endpoint_bocf_percent"                                                
 [76] "pain_participants_with_50_percent_decrease_from_baseline_in_average_24hr_pain_score_endpoint_bocf_n"                                                      
 [77] "pain_participants_with_50_percent_decrease_in_bpi_average_pain_score_endpoint_locf_percent"                                                               
 [78] "pain_participants_with_50_percent_decrease_in_bpi_average_pain_score_endpoint_locf_n"                                                                     
 [79] "pain_participants_with_a_50_percent_reduction_in_24_h_average_pain_bpi_sf_item_5_post_intervention_percent"                                               
 [80] "pain_participants_with_a_50_percent_reduction_in_24_h_average_pain_bpi_sf_item_5_post_intervention_n"                                                     
 [81] "pain_participants_with_reduction_of_50_percent_in_the_bpi_24_hour_average_pain_score_endpoint_percent"                                                    
 [82] "pain_participants_with_reduction_of_50_percent_in_the_bpi_24_hour_average_pain_score_endpoint_n"                                                          
 [83] "pain_ped_24_hour_recall_pain_score_50_percent_improvement_from_baseline_endpoint_percent"                                                                 
 [84] "pain_ped_24_hour_recall_pain_score_50_percent_improvement_from_baseline_endpoint_n"                                                                       
 [85] "pain_relief_50_percent_relief_of_ped_24_hr_morning_recall_scores_completers_only_15_weeks_within_trial_percent"                                           
 [86] "pain_relief_50_percent_relief_of_ped_24_hr_morning_recall_scores_completers_only_15_weeks_within_trial_n"                                                 
 [87] "pain_response_patients_with_50_percent_decrease_in_mean_weekly_pain_endpoint_percent"                                                                     
 [88] "pain_response_patients_with_50_percent_decrease_in_mean_weekly_pain_endpoint_n"                                                                           
 [89] "participants_with_a_50_percent_reduction_in_the_weekly_mean_of_the_24_hour_average_pain_score_diary_based_endpoint_percent"                               
 [90] "participants_with_a_50_percent_reduction_in_the_weekly_mean_of_the_24_hour_average_pain_score_diary_based_endpoint_n"                                     
 [91] "participants_with_a_50_percent_reduction_in_the_weekly_mean_of_the_24_hour_average_pain_score_on_the_11_point_nrs_endpoint_percent"                       
 [92] "participants_with_a_50_percent_reduction_in_the_weekly_mean_of_the_24_hour_average_pain_score_on_the_11_point_nrs_endpoint_n"                             
 [93] "participants_with_reduction_of_50_percent_in_the_bpi_24_hour_average_pain_score_post_intervention_percent"                                                
 [94] "participants_with_reduction_of_50_percent_in_the_bpi_24_hour_average_pain_score_post_intervention_n"                                                      
 [95] "participants_with_reduction_of_50_percent_in_the_bpi_modified_short_form_24_hour_average_pain_score_3_months_acute_phase_post_intervention_percent"       
 [96] "participants_with_reduction_of_50_percent_in_the_bpi_modified_short_form_24_hour_average_pain_score_3_months_acute_phase_post_intervention_n"             
 [97] "participants_with_reduction_of_50_percent_in_the_bpi_modified_short_form_24_hour_average_pain_score_6_months_continuation_phase_post_intervention_percent"
 [98] "participants_with_reduction_of_50_percent_in_the_bpi_modified_short_form_24_hour_average_pain_score_6_months_continuation_phase_post_intervention_n"      
 [99] "percentage_of_responders_participants_with_50_percent_improvement_in_pain_100_mm_vas_endpoint_percent"                                                    
[100] "percentage_of_responders_participants_with_50_percent_improvement_in_pain_100_mm_vas_endpoint_n"                                                          
[101] "responders_50_percent_or_greater_reduction_of_pain_from_baseline_0_10_nrs_endpoint_percent"                                                               
[102] "responders_50_percent_or_greater_reduction_of_pain_from_baseline_0_10_nrs_endpoint_n"

**Comment by softloud** [comment]
[subpain-export.csv](https://github.com/softloud/happypillpain/files/6336582/subpain-export.csv)

**Comment by softloud** [comment]
Couldn't sleep, so coding. 

![image](https://user-images.githubusercontent.com/13408706/115330708-8263bd00-a18c-11eb-85cc-4495f0ad0308.png)

So, we have _better_ numbers, but we are still a way's off. I suspect I'm not picking up on a measure. 

More data detective work :womans_hat: to do. 

ugh where are all the substantial pain studies going?

**Comment by softloud** [comment]
Meanwhile, I think we should have a think about _which_ outcomes we want to discuss in the upcoming manuscript. Perhaps focus on at most three? 

I think you'd have a more nuanced opinion, @HollieBirkinshaw, but I was thinking: pain, mood, and adverse.

Currently substantial pain is placeholder for adverse 'cause there's something buggy about the assumptions with adverse we need to figure out. 

Is likely something Tamar & Gav will have opinions about.

**Comment by softloud** [comment]
Column names are really tricky because of weird characters. :thinking:

Is there a chance it's easy for you to match the column number to the name?

![image](https://user-images.githubusercontent.com/13408706/115595847-51d86c00-a2cf-11eb-9e8c-4a03990f9095.png)

**Comment by softloud** [comment]
- adverse events baseline shouldn't be there

**Comment by softloud** [comment]
merged with #61
